pkgs/applications/editors/emacs/elisp-packages/recipes-archive-melpa.json

```diff
@@ -106852,35 +106852,6 @@
    "sha256": "1dfmlmmf13qzzfpmpc6lgwjhz8hnz5ys2fw4w3rrz6swfaa2vr53"
   }
  },
- {
```

Comment:
It looks like this is manually removed. You shouldn't manually remove anything since this file is auto-generated and it will be re-added on the next run.

The manually packaged derivations take precedence over the auto-generated ones anyway so this is essentially a no-op.